fortnitepy/party.py

```diff
@@ -1198,6 +1198,19 @@ def set_playlist(self, playlist: Optional[str] = None, *,
         key = 'Default:PlaylistData_j'
         return {key: self.set_prop(key, final)}
 
+      
+    def set_mnemonic(self, mnemonic: Optional[str] = None) -> Dict[str, Any]:
+        data = (self.get_prop('Default:PlaylistData_j'))['PlaylistData']
+
+        if playlist is not None:
+            data['playlistName'] = Playlist_PlaygroundV2
```

Comment:
Also, `Playlist_PlaygroundV2` is not defined. You probably meant to make this a string?

Reply:
yeah i did oops